lib/api/controllers/documentController.js

```diff
@@ -146,9 +146,13 @@ class DocumentController extends BaseController {
    * @returns {Promise<Object>}
    */
   mGet (request) {
-    const
-      ids = this.getBodyArray(request, 'ids'),
-      { index, collection } = this.getIndexAndCollection(request);
+    let ids;
+    if (request.input.body) {
```

Comment:
The following request is valid, but this test makes it fail:

`{"controller":"document", "action":"mGet", "ids": ["oh", "noes"], "body": {}, "index": "foo", "collection": "bar"}`